lib/request/index.js

```diff
@@ -0,0 +1,15 @@
+var runtime = require('postman-runtime'),
+    util = require('./util');
+
+const req = function (options, callback) {
+    var runner = new runtime.Runner();
+
+    const collection = util.convert(options.curl, callback);
```

Comment:
`util.convert` has an inconsistent return format.
- For successful processing, it is returning the data synchronously.
- For errors, it is calling the callbacks. Also which we didn't handled.

Please use a consistent return format. Since the `convert` method of the module is using a callback, we will have to use callback for all returning statements and handle it properly here.